package.json

```diff
@@ -1,21 +1,63 @@
 {
   "name": "@stoplight/types",
-  "version": "1.2.2",
-  "author": "Stoplight <support@stoplight.io>",
-  "license": "MIT",
+  "version": "0.0.0",
+  "description": "Common typings for the Stoplight ecosystem.",
+  "keywords": [],
   "main": "./index.d.ts",
   "types": "./index.d.ts",
-  "scripts": {
-    "build": "tsc --project tsconfig.json"
+  "sideEffects": false,
+  "homepage": "https://github.com/stoplightio/types",
+  "bugs": "https://github.com/stoplightio/types/issues",
+  "author": "Stoplight <support@stoplight.io>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stoplightio/types.git"
   },
-  "prettier": {
-    "printWidth": 100,
-    "singleQuote": true,
-    "trailingComma": "es5"
+  "license": "Apache-2.0",
+  "files": [
+    "**/*",
```

Comment:
How about
```suggestion
    "**/*.d.ts",
```

Reply:
Before we commit to this one.

> Side note - is linting / type checking working for anybody in these declaration files? If I rename a file or interface I get no error reporting anywhere to tell me (scary!).

cc @chohmann @XVincentX @wmhilton